components/button/README.md

```diff
@@ -198,7 +166,21 @@ The `d2l-button-add` element is for quickly adding new items inline (for example
 
 ## Floating Buttons [d2l-floating-buttons]
 
-See [floating buttons](https://github.com/BrightspaceUI/core/tree/main/components/button/floating-buttons.md).
+See [floating buttons](../../components/floating-buttons).
+
+## Accessibility
+
+Daylight buttons rely on standard button semantics to ensure a smooth experience for all assistive technologies, but there are a few interesting details to note:
+
+* If the button's context is implied by visual layout, then `description` can be used to add missing context
+  * Example: if multiple page sections have an Edit button relying on visual layout to indicate which section it edits, there could be extra information in the `description` to help differentiate the Edit buttons for non-sighted users
+
+* For icon buttons where there is no visible, `text` will be displayed in tooltip
```

Comment:
Is this what you meant?

```suggestion
* For icon buttons where there is no text visible, `text` will be displayed in tooltip
```

Reply:
Ahh, I mean "no visible label" but must have removed it in one of my edits 🤦‍♂️